humboldt-state-university-environmental-resources-engineering.csl

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Humboldt State University-Environmental Resources Engineering</title>
+    <title-short>hsu</title-short>
+    <id>http://www.zotero.org/styles/Humboldt State Univeristy-Environmental Resources Engineering</id>
+    <link href="http://www.zotero.org/styles/humboldt-state-university-environmental-resrouces-engineering" rel="self"/>
+    <link href="https://www2.humboldt.edu/engineering/resources/technical-communication" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/american-society-of-civil-engineers" rel="template"/>
+    <author>
+      <name>Daniel Burgett and Chase Cimina</name>
+    </author>
```

Comment:
This is acceptable, but it would be nicer to have:

```xml
    <author>
      <name>Daniel Burgett</name>
    </author>
    <author>
      <name>Chase Cimina</name>
    </author>
```